test/aria/widgets/wai/input/WaiInputTestSuite.js

```diff
@@ -28,7 +28,7 @@ Aria.classDefinition({
         this.addTests("test.aria.widgets.wai.input.multiSelect.MultiSelectTestCase");
         this.addTests("test.aria.widgets.wai.input.datePicker.DatePickerTestCase");
         this.addTests("test.aria.widgets.wai.input.dateField.DateFieldTestCase");
-        this.addTests("test.aria.widgets.wai.input.checkBox.WaiCheckBoxTestSuite");
-        this.addTests("test.aria.widgets.wai.input.radiobutton.WaiRadioButtonTestSuite");
```

Comment:
Tests which are no longer passing and which are not relevant should be removed (including test files), and not only commented in the test suite.